### PR TITLE
[GSW-463] feat: Calculate the Price Range of Liquidity

### PR DESCRIPTION
**Pull request by jinoosss**

### CODE REVIEW

File: packages/web/src/components/common/pool-selection-graph/PoolSelectionGraph.stories.tsx
Comment (khiemldk) [inline packages/web/src/components/common/pool-selection-graph/PoolSelectionGraph.stories.tsx:33]: it is redundant

File: packages/web/src/components/common/select-price-range-custom/SelectPriceRangeCustom.tsx
Comment (khiemldk) [inline packages/web/src/components/common/select-price-range-custom/SelectPriceRangeCustom.tsx:125]: i'm not sure in future this value will be change? can we add this as a constant value?.  Min Rate, MaxRate

File: packages/web/src/providers/gnoswap-service-provider/GnoswapServiceProvider.tsx
Comment (khiemldk) [inline packages/web/src/providers/gnoswap-service-provider/GnoswapServiceProvider.tsx:72]: I have an idea for process.env value. We can default this in constant and import this. To easy manage variable

File: packages/web/src/components/common/select-price-range-custom/SelectPriceRangeCustom.tsx
Comment (jinoosss) [reply packages/web/src/components/common/select-price-range-custom/SelectPriceRangeCustom.tsx:125]: Thanks for the suggestion. I think it would be better to manage it as a constant variable.

File: packages/web/src/providers/gnoswap-service-provider/GnoswapServiceProvider.tsx
Comment (jinoosss) [reply packages/web/src/providers/gnoswap-service-provider/GnoswapServiceProvider.tsx:72]: Are you suggesting to manage `DEFAULT_CHAIN_ID` as a constant variable?

Comment (khiemldk) [reply packages/web/src/providers/gnoswap-service-provider/GnoswapServiceProvider.tsx:72]: Yep.

**Review by khiemldk — DISMISSED** [review]

**Review by dongwon8247 — CHANGES_REQUESTED** [review]
Don't merge yet, it will be merged after khiemldk's ongoing work.

**Review by dongwon8247 — COMMENTED** [review]
@jinoosss Can you resolve conflicts?
@khiemldk Please give it a review before merging it

**Comment by jinoosss** [comment]
> @jinoosss Can you resolve conflicts? @khiemldk Please give it a review before merging it

I have resolved the conflict.

Also, due to a package manager update, I updated the dependency cache file.

**Review by khiemldk — DISMISSED** [review]

**Pull request merged**
